db/fixtures/miq_product_features.yml

```diff
@@ -1468,6 +1468,10 @@
     :description: Modify Dashboard
     :feature_type: admin
     :identifier: dashboard_admin
+  - :name: Tag
+    :description: Tag in Dashboard
+    :feature_type: control
+    :identifier: dashboard_tag
```

Comment:
Ok, I figured out why the tests are failing... Tag needs to be added either after dashboard_view/before dashboard_admin OR after the children of dashboard_admin.

As it is now, the children of dashboard_admin such as dashboard_add are moved are children of dashboard_tag.

Like this:

```
[3] pry(#<RSpec::ExampleGroups::MiqUserRole::TestingAllowsMethods>)> MiqProductFeature.find_by(:identifier=> "dashboard_add").parent
=> #<MiqProductFeature:0x00007fa574759538
 id: 1816,
 identifier: "dashboard_tag",
 name: "Tag",
 description: "Tag in Dashboard",
 feature_type: "control",
 protected: false,
 parent_id: 1814,
 created_at: Thu, 27 May 2021 18:21:08 UTC +00:00,
 updated_at: Thu, 27 May 2021 18:21:08 UTC +00:00,
 hidden: nil,
 tenant_id: nil>
```

I'll push a change to fix it.

Reply:
Ok, dashboard_admin now has their children back:  dashboard_add and dashboard_reset.  Tests should pass.

Reply:
Thanks @jrafanie.